alignak/acknowledge.py

```diff
@@ -96,3 +97,35 @@ def serialize(self):
                 'end_time': self.end_time, 'author': self.author, 'comment': self.comment,
                 'persistent': self.persistent
                 }
+
+    def get_and_start_brok(self, host_name, service_name=''):
+        """Get a start acknowledge brok
+
+        :param comment_type: 1 = host, 2 = service
+        :param host_name:
+        :param service_name:
+        :return: brok with wanted data
+        :rtype: alignak.brok.Brok
+        """
+        data = {'uuid': self.uuid}
+        data['host'] = host_name
+        if service_name != '':
+            data['service'] = service_name
```

Comment:
`data` should also contain the acknowledge specific parameters: sticky, persistent, author, comment, ...